src/rgw/rgw_civetweb.cc

```diff
@@ -111,7 +111,7 @@ void RGWCivetWeb::init_env(CephContext *cct)
   }
 
   env.set("REQUEST_METHOD", info->request_method);
-  env.set("REQUEST_URI", info->uri);
+  env.set("REQUEST_URI", info->request_uri); // get the full uri, we anyway handle abs uris later
```

Comment:
@theanalyst is this change needed? it's not clear to me what's the effect of this line